<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QDialog" name="NewGame">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="Versus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>63</height>
    </rect>
   </property>
   <property name="title">
    <string>Versus</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="VsHuman">
      <property name="text">
       <string>Human</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="VsIA">
      <property name="text">
       <string>Computer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="ReseauBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Reseau</string>
   </property>
   <widget class="QCheckBox" name="cbNetwork">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="TWNetwork">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="CreateGame">
     <attribute name="title">
      <string>Create</string>
     </attribute>
     <widget class="QPushButton" name="btnCreate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="JoinGame">
     <attribute name="title">
      <string>Join</string>
     </attribute>
     <widget class="QPushButton" name="listRefresh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnJoin">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listGame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="ComputOrHuman">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Human Player</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="Color">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>201</width>
     <height>63</height>
    </rect>
   </property>
   <property name="title">
    <string>Color</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="WhiteColor">
      <property name="text">
       <string>White</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BlackColor">
      <property name="text">
       <string>Black</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
